rts/motoko-rts/src/memory/ic.rs

```diff
@@ -88,17 +85,17 @@ impl Memory for IcMemory {
     }
 }
 
-/// Page allocation. Ensures that the memory up to, and including, the given pointer is allocated,
+/// Page allocation. Ensures that the memory up to, but excluding, the given pointer is allocated,
 /// with the slight exception of not allocating the extra page for address 0xFFFF_0000.
-/// Enforced precondition: `ptr` must be a pointer less than or equal to 0xFFFF_0000.
 #[inline(never)]
 unsafe fn grow_memory(ptr: u64) {
-    debug_assert_eq!(0xFFFF_0000, usize::MAX - WASM_PAGE_SIZE.as_usize() + 1);
+    debug_assert!(ptr <= 2 * u64::from(core::u32::MAX));
```

Comment:
This is not needed any more given line 93. Maybe I forgot to remove it from #3090.

Reply:
Done